src/hyspecppt/hppt/hppt_view.py

```diff
@@ -271,6 +274,18 @@ def validate_inputs(self, *dummy_args, **dummy_kwargs) -> None:
         print(self.sender)
         print("args: ", dummy_args, " kwargs: ", dummy_kwargs)
 
+    def set_values(self, values: dict[str, float]) -> None:
+        """Sets widget display based on the values dictionary
+
+        Args:
+            values (dict): a dictionary that contains
+            Ei, S2, alpha_p, plot_types values
```

Comment:
Can you check in the entire source code for S2 vs s2. I don't care which, but let's be consistent

Reply:
Setting all to upper S2.